sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/queries/sql/SqlQuery1.java

```diff
@@ -79,4 +82,21 @@ public SqlQuery1() {
   private RowCoder getBidRowCoder() {
     return ADAPTERS.get(Bid.class).getRowType().getRowCoder();
   }
+
+  public PCollection<? extends KnownSize> applyModel(PCollection<Row> rows) {
```

Comment:
would it make more sense to keep all the mapping from and to models in a single place, e.g. [`ModelAdaptersMapping`](https://github.com/apache/beam/blob/11ad97e42015dc497fb5cc3d38090e64f9c7418b/sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/model/sql/adapter/ModelAdaptersMapping.java), by adding a method to [`ModelFieldsAdapter`](https://github.com/apache/beam/blob/11ad97e42015dc497fb5cc3d38090e64f9c7418b/sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/model/sql/adapter/ModelFieldsAdapter.java), and maybe a utility method in [`ToRow`](https://github.com/apache/beam/blob/a13c7d9177c34017955cbfbfc29a7f4a698047c1/sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/model/sql/ToRow.java)

Reply:
I think it does make sense to add the reverse mappings to `ModelAdaptersMapping` and referenece that here. I'll update.